src/components/stories/TextSearch.js

```diff
@@ -15,37 +19,40 @@ if (typeof window == "object") {
   window.L10N.setBundle(require("../../../assets/panel/debugger.properties"));
 }
 
-function TextSearchFactory(options, { dir = "ltr", theme = "light" } = {}) {
+function TextSearchFactory(options, { dir = "ltr", theme = "dark" } = {}) {
   const themeClass = `theme-${theme}`;
   document.dir = dir;
   document.body.parentNode.className = themeClass;
 
   prefs.searchNav = true;
 
-  return dom.div(
-    {
-      className: "",
-      style: {
-        width: "calc(100vw - 100px)",
-        height: "calc(100vh - 100px)",
-        margin: "auto",
-        display: "flex",
-        "flex-direction": "row"
-      }
-    },
+  return Shortcuts(
+    {},
     dom.div(
       {
-        className: `search-bar ${themeClass}`,
-        dir,
+        className: "",
         style: {
-          width: "100vw",
-          "align-self": "center"
+          width: "calc(100vw - 100px)",
+          height: "calc(100vh - 100px)",
+          margin: "auto",
```

Comment:
my gut instinct is to move styles that are not calculated to a class, but it was like this before so...